django/db/migrations/autodetector.py

```diff
@@ -576,6 +591,134 @@ def generate_renamed_models(self):
                             self.old_model_keys.add((app_label, model_name))
                             break
 
+    def generate_moved_models(self):
+        """
+        Find any moved models, generate the operations for them, and remove
+        the old entry from the model lists. Must be run before other
+        model-level generation.
+        """
+        added_models = self.new_model_keys - self.old_model_keys
+
+        for app_label, model_name in sorted(added_models):
+            model_state = self.to_state.models[app_label, model_name]
+            model_fields_def = self.only_relation_agnostic_fields(model_state.fields)
+            removed_models = self.old_model_keys - self.new_model_keys
+            for rem_app_label, rem_model_name in removed_models:
+                if rem_app_label != app_label:
```

Comment:
It is better to reduce the levels of nesting. If you reverse the condition:
```suggestion
                if rem_app_label == app_label:
                    continue
```
then the rest of the function can be nested one level less. Similarly for the next `if` at line 615 (when this is written).